cmd/yggmail/main.go

```diff
@@ -102,14 +103,18 @@ func main() {
 		copy(sk, skBytes)
 	}
 	pk := sk.Public().(ed25519.PublicKey)
-	log.Printf("Mail address: %s@%s\n", hex.EncodeToString(pk), utils.Domain)
+	mailAddr_user := hex.EncodeToString(pk);
```

Comment:
FYI terminating semicolons are not canonical Go, please make sure to format any changes with `goimports` or `gofmt` etc.

Reply:
Also should probably just be `mailUser` or so?